README.md

```diff
@@ -94,5 +94,15 @@ If you have any suggestions about ways that we can improve how we use this tool,
 
 ## FAQ
 
+#### CSS Variables
 [How do I use CSS variables to customize
 the library?](https://pf4.patternfly.org/guidelines#variables)
+
+#### Browser Support
+PatternFly 4 is supported on the latest version of the following browsers:
```

Comment:
Should we talk about how many of the previous versions we support - I think you mentioned the latest two?

Reply:
Let's leave it at latest initially.